backend/proxy-server/src/server.js

```diff
@@ -2,7 +2,7 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const multer = require('multer');
 const cors = require('cors');
-const fetch = require('node-fetch');
+const { pushData } = require('./integrations/airtable');
```

Comment:
wouldn't it be better if this was named something that indicated a relationship to the integration itself ( for example, `pushAirtableData` )? This would make it more flexible when more integrations are implemented ( they could be named `pushInt1Data`, `pushInt2Data`, `pushIntNData` )

Reply:
@shinenelson I considered this. But I was also wondering why we need to have a repetition of names when pushData exists in /airtable/index.js file.

Reply:
Since it depends on how we write the integrations actually, it can swing in both ways. So for now, I kept it as if we only had airtable.